terraform/aws_uni_topology/main.tf

```diff
@@ -11,7 +11,7 @@ locals {
 }
 
 module "vpc" {
-  source = "../../../terraform/aws_vpc"
+  source = "../aws_vpc"
```

Comment:
I'm not sure if I was doing something wrong, but nothing worked at all without changing these paths.

Reply:
This is odd. Test harness seems to work on `master` currently...

Reply:
Checked, doesn't work on my end either. Created #45.